cmd/buildah/bud.go

```diff
@@ -26,8 +26,8 @@ func init() {
 	budDescription := `
   Builds an OCI image using instructions in one or more Dockerfiles.
 
-  If no arguments specified, it will assume the current working directory as
-  build context, which should contain the Dockerfile.`
+  If no are arguments specified, it will assume the current working directory
+  as build context, which should contain the Dockerfile.`
```

Comment:
Independent of this change, I find the sentence a bit hard to brain parse and it doesn't explain that it'll fail otherwise.

"If no arguments are specified, Buildah will use the current working directory as the build context and look for a Dockerfile. The build fails if no Dockerfile is present."

Reply:
Changed as suggested.